android/src/main/java/com/evollu/react/fcm/SendNotificationTask.java

```diff
@@ -213,10 +213,12 @@ protected Void doInBackground(Void... params) {
                 notification.setContentIntent(pendingIntent);
 
                 if (bundle.containsKey("android_actions")) {
-                    String[] actions = bundle.getString("android_actions").split(",");
-                    for (int a = 0; a < actions.length; a++) {
-                        String actionValue = actions[a].trim();
-                        Intent actionIntent = new Intent(mContext, intentClass);
+                    List<String> actions = bundle.getStringArrayList("android_actions");
+                    bundle.remove("android_actions");
```

Comment:
let's keep the bundle as is

Reply:
@evollu okay, so what the way to keep `android_actions` and don't come across parsing error? It happened when parsing the Intent in `FIRMessagingModule.java`: it can't parse `ArrayList`.

Reply:
hmm..ok